config/seata-config-raft/src/main/java/org/apache/seata/config/raft/RaftConfigurationClient.java

```diff
@@ -0,0 +1,690 @@
+                                LOGGER.info("The configuration version: {} of the server is lower than the current configuration: {} , it may be expired configuration.", version, CONFIG_VERSION.get());
+                                throw new RetryableException("Expired configuration!");
+                            } else {
+                                CONFIG_VERSION.set(version);
+                                return configMap;
+                            }
+                        } else {
+                            throw new RetryableException(configDataResponse.getErrMsg());
+                        }
+                    } catch (JsonProcessingException e) {
+                        LOGGER.error(e.getMessage(), e);
+                    }
+                }
+            } catch (IOException e) {
+                throw new RetryableException(e.getMessage(), e);
+            }
+        }
+        return null;
+    }
+
+    protected static void startQueryMetadata() {
+        if (REFRESH_METADATA_EXECUTOR == null) {
+            synchronized (INIT_ADDRESSES) {
+                if (REFRESH_METADATA_EXECUTOR == null) {
+                    REFRESH_METADATA_EXECUTOR = new ThreadPoolExecutor(1, 1, 0L, TimeUnit.MILLISECONDS,
+                            new LinkedBlockingQueue<>(), new NamedThreadFactory("refreshMetadata", 1, true));
+                    REFRESH_METADATA_EXECUTOR.execute(() -> {
+                        long metadataMaxAgeMs = FILE_CONFIG.getLong(ConfigurationKeys.CLIENT_METADATA_MAX_AGE_MS, 30000L);
+                        long currentTime = System.currentTimeMillis();
+                        while (!CLOSED.get()) {
+                            try {
+                                // Forced refresh of metadata information after set age
+                                boolean fetch = System.currentTimeMillis() - currentTime > metadataMaxAgeMs;
+                                String clusterName = RAFT_CLUSTER;
+                                if (!fetch) {
+                                    fetch = watch();
+                                }
+                                // Cluster changes or reaches timeout refresh time
+                                if (fetch) {
+                                    for (String group : METADATA.groups(clusterName)) {
+                                        try {
+                                            acquireClusterMetaData(clusterName, group);
+                                        } catch (Exception e) {
+                                            // prevents an exception from being thrown that causes the thread to break
+                                            if (e instanceof RetryableException) {
+                                                throw e;
+                                            } else {
+                                                LOGGER.error("failed to get the leader address,error: {}", e.getMessage());
```

Comment:
Use   LOGGER.error("failed to get the leader address,error: {}", e.getMessage(),e);